app/src/main/res/values/styles_services.xml

```diff
@@ -1,32 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- YouTube -->
-    <style name="LightTheme.YouTube" parent="LightTheme" tools:keep="@style/LightTheme_YouTube" />
+    <style name="LightTheme.YouTube" parent="LightTheme" tools:keep="@style/LightTheme_YouTube">
+        <item name="colorPrimaryDark">@color/light_youtube_primary_color</item>
+    </style>
 
-    <style name="DarkTheme.YouTube" parent="DarkTheme" tools:keep="@style/DarkTheme_YouTube" />
+    <style name="DarkTheme.YouTube" parent="DarkTheme" tools:keep="@style/DarkTheme_YouTube">
+        <item name="colorPrimaryDark">@color/dark_youtube_primary_color</item>
+    </style>
 
-    <style name="BlackTheme.YouTube" parent="BlackTheme" tools:keep="@style/BlackTheme_YouTube" />
+    <style name="BlackTheme.YouTube" parent="BlackTheme" tools:keep="@style/BlackTheme_YouTube">
+        <item name="colorPrimaryDark">@color/dark_youtube_primary_color</item>
+    </style>
 
     <!-- SoundCloud -->
     <style name="LightTheme.SoundCloud" parent="LightTheme" tools:keep="@style/LightTheme_SoundCloud">
         <item name="colorPrimary">@color/light_soundcloud_primary_color</item>
-        <item name="colorPrimaryDark">@color/light_soundcloud_dark_color</item>
+        <item name="colorPrimaryDark">@color/light_soundcloud_primary_color</item>
```

Comment:
(reminder for me to make sure the status bar is still darker in dark theme)

Reply:
Yes, there is a problem:
<img src="https://user-images.githubusercontent.com/36421898/156531908-f7b3b6d8-ebe9-4db8-8234-02e565824cea.jpg" height="300px"/>

Reply:
I don't see why that needed to be changed in the first place? 🤔

Reply:
Is this a problem I should work on or just a sidenote.
Because if it's a problem I should work on, I don't know what is meant with the above picture. Everything works fine when I install the app:
<img src="https://user-images.githubusercontent.com/40789489/156818982-3b42b0d6-9285-4073-ae93-53cfa09bcbde.png" height=400 />
This is exactly the same behavior as with the current release.

Does this only occur on a specific android version?

Reply:
@Stypox Should be fixed now 😄

Reply:
Can confirm